03_launch_mgmt_cluster.sh

```diff
@@ -158,6 +160,7 @@ IRONIC_FAST_TRACK=true
 RESTART_CONTAINER_CERTIFICATE_UPDATED="${RESTART_CONTAINER_CERTIFICATE_UPDATED}"
 IRONIC_RAMDISK_SSH_KEY=${SSH_PUB_KEY_CONTENT}
 IRONIC_USE_MARIADB=${IRONIC_USE_MARIADB:-false}
+${inspector_default}
```

Comment:
shouldn't we remove line 157 `IRONIC_INSPECTOR_ENDPOINT`

Reply:
We cannot do it without breaking the current state of things (and the release branches).

Reply:
ah right !